crates/ruff_linter/src/rules/flake8_pyi/rules/unnecessary_literal_union.rs

```diff
@@ -31,14 +31,18 @@ pub struct UnnecessaryLiteralUnion {
     members: Vec<String>,
 }
 
-impl Violation for UnnecessaryLiteralUnion {
+impl AlwaysFixableViolation for UnnecessaryLiteralUnion {
     #[derive_message_formats]
     fn message(&self) -> String {
         format!(
             "Multiple literal members in a union. Use a single literal, e.g. `Literal[{}]`",
             self.members.join(", ")
         )
     }
+
+    fn fix_title(&self) -> String {
+        format!("Replace Literal unions with a single Literal.")
```

Comment:
Nit: "Replace with a single `Literal`" would better match our style for these messages.